lib/mobility-core/src/Kernel/Storage/Esqueleto/Transactionable.hs

```diff
@@ -104,4 +116,26 @@ runNoTransactionIO logEnv dbEnv (SqlDB run) = do
         SqlDBEnv
           { currentTime = now
           }
-  runLoggerIO logEnv $ runSqlPoolNoTransaction (runReaderT run sqlDBEnv) dbEnv.connPool Nothing
+  runLoggerIO logEnv $ do
+    res <- try @_ @SomeException $ runSqlPoolNoTransaction (runReaderT run sqlDBEnv) dbEnv.connPool Nothing
+    fromEitherOrM res $ \e -> do
+      let res' = transformException e
+      logTagError "DB QUERY" $ "Transformed ERROR response: " <> show res'
+      case res' of
+        ET.DBError (ET.SQLError (ET.PostgresError (ET.PostgresSqlError "" ET.PostgresFatalError "" "" ""))) _ -> do
```

Comment:
this case will match only when all these fields will be empty
```
       sqlState = ""
    , sqlErrorMsg = ""
    , sqlErrorDetail = ""
    , sqlErrorHint  = ""
```
Is it really ever happens?
Maybe you meant this:
`ET.PostgresSqlError _ ET.PostgresFatalError _ _ _)`

Reply:
No actually this is the exact case when db connection expires. 
```
beckn-rider-dashboard-master-2ad616-76fff5645b-vj4zf beckn-rider-dashboard-master {"log":"2024-12-17 11:02:50.314332966 UTC ERROR> @beckn-rider-dashboard-master-2ad616-76fff5645b-vj4zf [requestId-9a895842-4e09-416c-ba91-d63c9c689ceb, DB QUERY] |> Transformed ERROR response: DBError (SQLError (PostgresError (PostgresSqlError {sqlState = \"\", sqlExecStatus = PostgresFatalError, sqlErrorMsg = \"\", sqlErrorDetail = \"\", sqlErrorHint = \"\"}))) \"SqlError {sqlState = \\\"\\\", sqlExecStatus = FatalError, sqlErrorMsg = \\\"\\\", sqlErrorDetail = \\\"\\\", sqlErrorHint = \\\"\\\"}\""}
```